.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug report
+title: "[Bug] "
+description: Problems and issues with code of Linkis
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting the problem!
+        Please make sure what you are reporting is a bug with reproducible steps. To ask questions
+        or share ideas, pleae post on our [Discussion page](https://github.com/apache/incubator-linkis/discussions) instead.
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please make sure to search in the [issues](https://github.com/apache/incubator-linkis/issues) first to see
+        whether the same issue was reported already.
+      options:
+        - label: >
+            I searched the [issues](https://github.com/apache/incubator-linkis/issues) and found no similar
+            issues.
+          required: true
+
+  - type: dropdown
+    attributes:
+      label: Linkis Component
+      description: |
+        What component are you using? Linkis has many subprojects, please make sure to choose the component that
+        you found the bug.
+      multiple: true
+      options:
```

Comment:
I think these modules listed here aren't exactly correct, Could you help to confirm this?  @peacewong

Reply:
Yes, it should only contain the following modules:
```
linkis-commons
linkis-cg-entrance
linkis-cg-manager
linkis-cg-engineConnplugin
linkis-cg-engineConnManager
linkis-cg-client
linkis-extensions
linkis-orchestrator
linkis-public-enhancements
linkis-service-gateway
linkis-service-discovery
web
```